pkg/jaeger/store/binary_tags.go

```diff
@@ -0,0 +1,136 @@
+package store
+
+import (
+	"encoding/base64"
+	"fmt"
+	"strings"
+
+	"github.com/jaegertracing/jaeger/model"
+)
+
+const VALUE_TYPE_BINARY = "__ValueType_BINARY__"
+const VALUE_TYPE_BINARY_LEN = len(VALUE_TYPE_BINARY)
```

Comment:
Can we follow data URI spec[1] which is been widely used in the web world? May be following a standard would help us to pitch this change in the opentelemetry upstream without much hassle.

e.g.

```
data:application/octet-stream; base64,AAAwOQ==
```

Might look lengthy, but it follows the standard :)

[1] https://www.rfc-editor.org/rfc/rfc2397#section-2